integrations/amazon_bedrock/tests/test_generator.py

```diff
@@ -236,21 +236,20 @@ def test_get_model_adapter_model_family_over_auto_detection():
 
 class TestAnthropicClaudeAdapter:
     def test_default_init(self) -> None:
-        adapter = AnthropicClaudeAdapter(model_kwargs={}, max_length=100)
+        adapter = AnthropicClaudeAdapter(model_kwargs={})
         assert adapter.use_messages_api is True
 
     def test_use_messages_api_false(self) -> None:
-        adapter = AnthropicClaudeAdapter(model_kwargs={"use_messages_api": False}, max_length=100)
+        adapter = AnthropicClaudeAdapter(model_kwargs={"use_messages_api": False})
         assert adapter.use_messages_api is False
 
 
 class TestAnthropicClaudeAdapterMessagesAPI:
     def test_prepare_body_with_default_params(self) -> None:
-        layer = AnthropicClaudeAdapter(model_kwargs={}, max_length=99)
+        layer = AnthropicClaudeAdapter(model_kwargs={})
         prompt = "Hello, how are you?"
         expected_body = {
             "messages": [{"role": "user", "content": "Hello, how are you?"}],
-            "max_tokens": 99,
```

Comment:
Let's add this back and just set `max_tokens=99` in the init of `AnthropicClaudeAdapter`